Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py

```diff
@@ -30,7 +30,7 @@
     YARARetroHunting,
 )
 
-VERSION = f"v{get_pack_version()}" or "v2.7.0"
+VERSION = f"v{get_pack_version()}" or "v2.7.1"
```

Comment:
```suggestion
VERSION = f"v{get_pack_version() or '2.7.1'}"
```
Otherwise this is always true as the `v` is always included